packages/react/src/modal/ModalDialog.tsx

```diff
@@ -56,22 +56,22 @@ export const ModalDialog = ({
 					},
 				}}
 			>
-				<ModalTitle id={titleId}>{title}</ModalTitle>
-				<Box>{children}</Box>
-				{actions ? (
-					<Box paddingTop={1} css={{ marginTop: 'auto' }}>
-						{actions}
-					</Box>
-				) : null}
 				<Button
 					variant="text"
 					aria-label="Close modal"
 					onClick={closeHandler}
 					iconAfter={CloseIcon}
-					css={{ order: -1, alignSelf: 'flex-end' }}
+					css={{ alignSelf: 'flex-end' }}
 				>
 					Close
 				</Button>
+				<ModalTitle id={titleId}>{title}</ModalTitle>
```

Comment:
Not going to pass `title` to `aria-label`?

Reply:
Like we're now doing with `MainNavDialog`?

Reply:
No, because `aria-labelledby` overrides it and we focus the heading. The nav dialog one is only because we don't have a heading

Reply:
Ahh, that's right. Nice, thanks.